less/components/organize.less

```diff
@@ -0,0 +1,189 @@
+            padding: 2px;
+        }
+
+        &.node-hide .node {
+            opacity: 0.65;
+            border: 2px dashed #788896;
+            padding: 2px;
+            color: transparent;
+            background-color: transparent;
+
+            .icon {
+                fill: transparent;
+            }
+
+            input {
+                visibility: hidden;
+            }
+        }
+
+        &.node-empty > .node-parent > .node {
+            background-color: #ffc422;
+            border: 2px dashed #ff5722;
+            padding: 2px;
+        }
+    }
+
+    .node-parent {
+        display: flex;
+    }
+
+    .node {
+        display: flex;
+        align-items: center;
+        cursor: pointer;
+        padding: 4px;
+        color: @organize-node-fg;
+        background-color: @organize-node-bg;
+        margin-top: 10px;
+        height: 31px;
+        max-width: 100%;
+
+        & * {
+            // Stop children from spamming dragenter and dragleave events.
+            // https://stackoverflow.com/a/18582960
+            pointer-events: none;
+        }
+
+        & > a {
+            pointer-events: auto;
+
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            padding: 8px 6px;
+        }
+
+        & > input {
+            pointer-events: auto;
+
+            margin-right: 4px;
+
+            &, &::placeholder {
+                height: 30px;
+                font-size: 14px;
+                line-height: 1.5;
+            }
+        }
+
+        .icon {
+            fill: @organize-node-fg;
+            margin: 0;
+        }
+
+        .icon.symbol-icon {
+            margin-right: 12px;
+        }
+
+        .text {
+            font-size: 1.2em;
+            margin-right: 6px;
+
+            text-overflow: ellipsis;
+            white-space: nowrap;
+            overflow: hidden;
+        }
+
+        .icon.edit-icon {
+            font-size: 1em;
+        }
+
+        .drag-handle {
+            font-size: 1.0em;
+            margin-left: 6px;
+        }
+
+        &.root {
+            width: 200px;
+        }
+
+        &.trash {
+            width: 80px;
+            height: 80px;
+
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            .icon {
+                font-size: 3em;
+            }
+        }
+    }
+
+    .node-input {
+        display: flex;
+        padding: 4px;
+        margin-top: 10px;
+        color: @organize-node-fg;
+        background-color: @organize-node-bg;
+
+        .icon {
+            fill: @organize-node-fg;
+            margin-left: 0;
+            margin-right: 12px;
+            font-size: 2.0em;
+        }
+
+        .btn {
+            color: @organize-node-fg;
+        }
+    }
+
+    .form-control {
+        border-radius: 2px;
+        border-width: 2px;
+    }
+
+    p.error {
+        margin-top: @spacer/2;
+        color: #a94442;
+    }
+}
```

Comment:
I'm not sure where this popped in, but we have a horizontal scrollbar on all window widths I've tried. This needs to be fixed before merge (happy to help with this if it's being a pain to find! I find CSS relaxing).